pages/blog/caching-data-with-dataloader.mdx

```diff
@@ -0,0 +1,137 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Caching data with DataLoader',
+  author: 'giladtidhar',
+  tags: ['cache', 'data', 'servers', 'graphql'],
+  date: '2022-1-12',
+  description: 'How to use DataLoader to cache data',
+  image: '/blog-assets/caching-data-with-dataloader/cover.png'
+}
+export default withArticle({ ...meta })
+
+# Caching
+
+Caching data is an important thing, it could be the difference between a slow wasteful server and a fast efficient server.
+
+But, what is caching?
+A cache is a place in memory for important data, which can be delivered to the client faster than the regular procedure.  
+The goal of caching is to improve performance where it’s needed the most.
+
+## how is caching faster?
+
+Using a cache for important data is faster because of a few reasons:
+
+- A cache is usually not stored in storage, but in memory, which is a few times faster.
```

Comment:
```suggestion
- A cache is usually not stored on disk or database but in memory, which is faster.
```

Reply:
I think it should say something along the line of "a lot more faster" because if this article is for people who are new to this subject, just saying "faster" might make them not implement, might not be worth the trouble of implementing if it's just "faster"

Reply:
Agree 👍🏼